### ISSUE TITLE
Uncaught TypeError: Cannot read property 'onDidChange' of null

### ISSUE THREAD

**Issue by manuhdez** [atom/teletype#210]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.22.1 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13.1
**Thrown From**: [teletype](https://github.com/atom/teletype) package 0.1.2

### Stack Trace

Uncaught TypeError: Cannot read property 'onDidChange' of null

```
At /Users/manu_hdez/.atom/packages/teletype/lib/portal-list-component.js:14

TypeError: Cannot read property 'onDidChange' of null
    at /packages/teletype/lib/portal-list-component.js:14:57)
    at render (/packages/teletype/node_modules/etch/lib/render.js:17:25)
    at updateChildren (/packages/teletype/node_modules/etch/lib/patch.js:108:36)
    at patch (/packages/teletype/node_modules/etch/lib/patch.js:18:9)
    at updateSync (/packages/teletype/node_modules/etch/lib/component-helpers.js:116:20)
    at /packages/teletype/node_modules/etch/lib/component-helpers.js:72:7
    at DefaultScheduler.performUpdates (/packages/teletype/node_modules/etch/lib/default-scheduler.js:48:34)
```

### Commands

```
```

### Non-Core Packages

```
autoprefixer 3.7.1 
emmet 2.4.3 
pigments undefined 
sass-autocompile 0.13.3 
teletype 0.1.2 
```

**Comment by as-cii** [comment]
Thanks for the report, @manuhdez. 💯 

I am working on a pull-request to fix this issue right now, so stay tuned for an update!

**Comment by as-cii** [comment]
The fix for this issue has just been published on teletype v0.1.3. Thanks again for the report!
